packages/lesswrong/server/utils/feedUtil.ts

```diff
@@ -148,9 +148,14 @@ export async function mergeFeedQueries<SortKeyType>({limit, cutoff, offset, subq
   // Apply limit
   const withLimitApplied = _.first(bothResultKinds, limit);
   
+  // Find the last result that wasn't numerically positioned (after the limit
+  // is applied), and get its sortKey to use as the page cutoff
+  const nonNumericallyPositionedResults = _.filter(withLimitApplied, r => !_.some(numericallyPositionedResults, r2=>r===r2));
```

Comment:
Just fyi I think `_.difference` accomplishes the same thing, but this is fine too.